src/viam/ur/module/ur_arm.cpp

```diff
@@ -270,9 +270,10 @@ URArm::URArm(Model model, const Dependencies& deps, const ResourceConfig& cfg) :
     current_state_->keep_alive_thread = std::thread(&URArm::keep_alive, this);
     VIAM_SDK_LOG(info) << "URArm constructor end";
 }
-
+int rando_counter = 0;
```

Comment:
testing?

Reply:
ah yeah this was testing, had another round of cleanup to go through when I first put up the pr